src/cmem/ActivityControl/SilkActivityControl.tsx

```diff
@@ -286,9 +300,24 @@ export function useSilkActivityControl({
                 />
             </>
         ) : (
-            <>{label}</>
+            label
         );
 
+    const timerExecutionMessage =
+        (activityStatus?.startTime || activityStatus?.queueTime) && activityStatus.statusName !== "Finished" ? (
+            <ElapsedDateTimeDisplay
+                includeSeconds
+                dateTime={
+                    (activityStatus.statusName === "Running"
+                        ? activityStatus?.startTime
+                        : activityStatus.statusName === "Waiting"
+                        ? activityStatus.queueTime
+                        : new Date().toISOString())!
```

Comment:
This probably makes no sense, since it will always display "0 seconds" if the status is neither `Runing` nor `Waiting`. Please either use the `startTime` or no timer at all.